unicorecmsmama/tests/test_views.py

```diff
@@ -86,31 +86,6 @@ def localise_logo(locale, logo_text):
             resp = self.app.get('/?_LOCALE_=%s' % locale, status=200)
             return resp.body
 
-        # check default locale
-        resp = self.app.get('/', status=200)
-        self.assertTrue(
-            '<div id="banner">Advice from experts and parents</div>' in
-            resp.body)
-
-        # check locale without translation
-        self.assertTrue(
-            '<div id="banner">Advice from experts and parents</div>' in
-            localise_logo('eng_GB', None))
-        self.assertTrue(
-            '<div id="banner">Advice from experts and parents</div>' in
-            localise_logo('eng_UK', None))
-        self.assertTrue(
-            '<div id="banner">Advice foo</div>' in
-            localise_logo('eng_GB', 'Advice foo'))
-
-        # check locale with translation
-        self.assertTrue(
-            '<div id="banner">Saran dari pakar dan orang tua</div>' in
-            localise_logo('ind_ID', None))  # falls back to translation files
-        self.assertTrue(
-            '<div id="banner">Advice foo</div>' in
-            localise_logo('ind_ID', 'Advice foo'))
```

Comment:
Could of comments:

* We can delete the `def localise_logo` function.
* We should still call `resp = self.app.get('/', status=200)` and assert something about the response (just that the home page looks sane since the test is called `test_homepage_page`).